services/src/main/java/org/keycloak/services/resources/admin/permissions/RealmPermissions.java

```diff
@@ -197,5 +197,18 @@ public void requireViewAuthenticationFlows() {
         }
     }
 
+    @Override
+    public void requireViewPerClientConfigDescription() {
+        if (!(canViewRealm() || root.hasOneAdminRole(AdminRoles.QUERY_CLIENTS))) {
+            throw new ForbiddenException();
+        }
+    }
+
+    @Override
+    public void requireViewClientAuthenticatorProviders() {
```

Comment:
Same thing here. I think we should keep `requireViewClientAuthenticatorProviders`.